include/range/v3/view/iota.hpp

```diff
@@ -524,16 +543,17 @@ namespace ranges
                 detail::take_exactly_view_<iota_view<Val>, true>>
             ints_fn::operator()(Val from, Val to) const
             {
-                return {iota_view<Val>{from}, detail::iota_minus_(to, from)};
+                return {iota_view<Val>{from}, detail::ints_distance_(from, to)};
             }
 
             struct closed_ints_fn
             {
                 template<typename Val,
                     CONCEPT_REQUIRES_(Integral<Val>())>
+                RANGES_DEPRECATED("view::closed_ints is deprecated; use view::closed_indices instead!")
                 detail::take_exactly_view_<iota_view<Val>, true> operator()(Val from, Val to) const
                 {
-                    return {iota_view<Val>{from}, detail::iota_minus_(to, from) + 1};
+                    return {iota_view<Val>{from}, detail::ints_distance_(from, to) + 1};
```

Comment:
Yep. I remember now why I used difference type, it's because `take_exactly` takes a difference type, so that one is the one in which overflow must be prevented. 

I think I should write a comment because I did it 2 days ago and already forgot why. . . sigh.